en/get-started.md

```diff
@@ -0,0 +1,735 @@
+        ```
+        NAME:   tidb-operator
+        LAST DEPLOYED: Thu May 28 15:17:38 2020
+        NAMESPACE: tidb-admin
+        STATUS: DEPLOYED
+
+        RESOURCES:
+        ==> v1/ConfigMap
+        NAME                   DATA  AGE
+        tidb-scheduler-policy  1     0s
+
+        ==> v1/Deployment
+        NAME                     READY  UP-TO-DATE  AVAILABLE  AGE
+        tidb-controller-manager  0/1    1           0          0s
+        tidb-scheduler           0/1    1           0          0s
+
+        ==> v1/Pod(related)
+        NAME                                      READY  STATUS             RESTARTS  AGE
+        tidb-controller-manager-6d8d5c6d64-b8lv4  0/1    ContainerCreating  0         0s
+        tidb-controller-manager-6d8d5c6d64-b8lv4  0/1    ContainerCreating  0         0s
+
+        ==> v1/ServiceAccount
+        NAME                     SECRETS  AGE
+        tidb-controller-manager  1        0s
+        tidb-scheduler           1        0s
+
+        ==> v1beta1/ClusterRole
+        NAME                                   CREATED AT
+        tidb-operator:tidb-controller-manager  2020-05-28T22:17:38Z
+        tidb-operator:tidb-scheduler           2020-05-28T22:17:38Z
+
+        ==> v1beta1/ClusterRoleBinding
+        NAME                                   ROLE                                               AGE
+        tidb-operator:kube-scheduler           ClusterRole/system:kube-scheduler                  0s
+        tidb-operator:tidb-controller-manager  ClusterRole/tidb-operator:tidb-controller-manager  0s
+        tidb-operator:tidb-scheduler           ClusterRole/tidb-operator:tidb-scheduler           0s
+        tidb-operator:volume-scheduler         ClusterRole/system:volume-scheduler                0s
+
+
+        NOTES:
+        1. Make sure tidb-operator components are running
+           kubectl get pods --namespace tidb-admin -l app.kubernetes.io/instance=tidb-operator
+        2. Install CRD
+           kubectl apply -f https://raw.githubusercontent.com/pingcap/tidb-operator/master/manifests/crd.yaml
+           kubectl get customresourcedefinitions
+        3. Modify tidb-cluster/values.yaml and create a TiDB cluster by installing tidb-cluster charts
+           helm install tidb-cluster
+        ```
+
+        Confirm that the TiDB Operator components are running with this command:
+
+        {{< copyable "shell-regular" >}}
+
+        ```shell
+        kubectl get pods --namespace tidb-admin -l app.kubernetes.io/instance=tidb-operator
+        ```
+
+        Expected output:
+        ```
+        NAME                                       READY   STATUS    RESTARTS   AGE
+        tidb-controller-manager-6d8d5c6d64-b8lv4   1/1     Running   0          2m22s
+        tidb-scheduler-644d59b46f-4f6sb            2/2     Running   0          2m22s
+        ```
+
+        As soon as all pods are in "Running" state, proceed to the next step.
+
+2. Install TiDB Operator CRDs:
```

Comment:
This should be done before the TiDB Operator installation.

Reply:
OK

Reply:
I will fix this in a separate commit.